app/javascript/controllers/autocomplete_controller.js

```diff
@@ -7,13 +7,21 @@ export default class extends Controller {
   static targets = ['field', 'suggestionTemplate', 'emptyResultTemplate', 'lastOptionTemplate']
 
   connect () {
+    // Prevent Enter key from submitting the form
+    this.fieldTarget.addEventListener('keypress', function (e) {
+      if (e.key === 'Enter') {
+        e.preventDefault()
+      }
+    })
+
```

Comment:
@nicktiberi Is this somehow "scoped" to the autocomplete? If my cursor is focused someplace else on the page, this _shouldn't_ pick it up, right?

Reply:
Yep, it is scoped to `fieldTarget` aka the creators search `<input>` tag, so it should not affect other fields on the page. If your cursor is focused on any other "normal" element, hitting Enter will produce the default action of attempting to submit the form.